src/main/java/com/opentable/kafka/builders/KafkaBuilderFactoryBean.java

```diff
@@ -67,4 +71,36 @@
         return originalMap;
     }
 
+
+    public KafkaConsumerBuilder<?, ?> consumerBuilder() {
```

Comment:
Why are these not K,V. Please fix. This worked before.

Reply:
see comment